instruqt-tracks/terraform-cloud-aws-v2/03-centralize-deployments/check-workstation

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -l
+
+# Create /tmp/skip-check to disable this check
+if [ -f /tmp/skip-check ]; then
+    rm /tmp/skip-check
+    exit 0
+fi
+
+cd /root/hashicat-aws
```

Comment:
Later the construct `/root/hashicat-$CLOUD_ENV` is used, should it be used here as well?

Reply:
yes.